scipy/spatial/setup.py

```diff
@@ -78,8 +78,16 @@ def configuration(parent_package='', top_path=None):
     config.add_extension('_voronoi',
                          sources=['_voronoi.c'])
 
-    config.add_extension('_hausdorff',
-                         sources=['_hausdorff.c'])
+    try:
+        import pythran
+        import os
+        if os.name == 'nt':
+            raise ImportError()
+        config.ext_modules.append(pythran.dist.PythranExtension(
+            'scipy.spatial._hausdorff', ["scipy/spatial/_hausdorff.py"]))
```

Comment:
so Pythran would just replace the Cython implementation under the hood if available? Just want to make sure it isn't intercepting the Python interface, which has other logic checks not available in the Cython file.

Reply:
Yeah, that's the idea.